README.md

```diff
@@ -5,6 +5,17 @@ Rust bindings to the CoinOR CBC MILP Solver using the C API.
 Tested on Debian 10, AMD64, coinor-libcbc3 2.9.9+repack1-1.
 For more details on installing the `libCbc` dependencies, [see below](#prerequisites-installing-cbc-library-files).
 
+## ⚠️ Important warning
+
+By default, the cbc library is not thread safe, and this crate does not prevent manipulating multiple models in parallel.
+This means that by default, this crate **violates rust memory safety rules**.
```

Comment:
```suggestion
You [should obtain a thread-safe version of `cbc`](https://github.com/KardinalAI/coin_cbc/pull/7#issuecomment-786189288).
Otherwise, until this is permanently fixed in `cbc`, this crate by default **violates rust memory safety rules**.
```